src/utils/dateUtils.js

```diff
@@ -0,0 +1,48 @@
+export function formatDate(dateString) {
+  const date = new Date(dateString);
+  const year = date.getFullYear().toString().substr(2, 2);
+  const month = (date.getMonth() + 1).toString().padStart(2, '0');
+  const day = date.getDate().toString().padStart(2, '0');
+
+  return `${year}.${month}.${day}`;
+}
+
+export function calculateTimePassed(createdAt) {
+  const createdDate = new Date(createdAt);
+  const now = new Date();
+  const differenceInSeconds = Math.floor((now - createdDate) / 1000);
+  const differenceInMinutes = Math.floor(differenceInSeconds / 60);
+  const differenceInHours = Math.floor(differenceInMinutes / 60);
+  const differenceInDays = Math.floor(differenceInHours / 24);
+  const differenceInMonths = Math.floor(differenceInDays / 30);
+  const differenceInYears = Math.floor(differenceInMonths / 12);
+
+  if (differenceInSeconds < 120) {
```

Comment:
2분을 표현하고 싶으셨다면 아래 두가지 제안대로 해보는건 어떨까요?
```suggestion
  if (differenceInSeconds < 2 * 60) {
```
- 위처럼 작성하면 2분을 의미함을 명시적으로 읽을 수 있습니다.

```suggestion
  if (differenceInMinutes < 2) {
```
- 위처럼 작성하면 분 단위를 비교하고 있다는 것을 명시적으로 알 수 있습니다.

시간을 표시할때 직접적인 계산 결과를 기재해두는 것보다 시간임을 유추할 수 있게 시간 단위를 계산식으로 표현해두는게 훨씬 가독성이 좋습니다.
- 이유: 연산결과로는 어떤 숫자를 의미하는건지 유추하기 어렵기 때문

Reply:
단순히 피그마에 적혀있는대로 요구사항만을 적용하다보니 명시적인 작성법을 놓치고 있었네요 .. 앞으론 주의하면서 작성하도록 하겠습니다 !